polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -315,22 +323,25 @@ class FrameInner extends PureComponent<CombinedProps, State> {
                 id={APP_FRAME_MAIN}
                 data-has-global-ribbon={Boolean(globalRibbon)}
               >
-                <div
-                  className={classNames(
-                    styles.Content,
-                    hasDynamicTopBar && styles['Content-TopBarAndReframe'],
-                  )}
-                >
-                  {hasDynamicTopBar ? (
+                {hasDynamicTopBar ? (
+                  <Scrollable
+                    scrollbarWidth="thin"
```

Comment:
The scrollbar is currently cut off by the rounded corners of the frame, but I will take a look at that in https://github.com/Shopify/polaris-backlog/issues/1379 when I restyle the scrollbar